assets/js/common-components/lineItems.js

```diff
@@ -3,7 +3,8 @@
 storefrontApp.component('vcLineItems', {
     templateUrl: "themes/assets/js/common-components/lineItems.tpl.liquid",
     bindings: {
-        order: '<'
+        order: '<',
+        checkoutCompletePage: '<'
```

Comment:
I think we should ask @godothor to fix design and make line items look the same on different pages